core/src/utils/pluginDetails.js

```diff
@@ -63,61 +63,77 @@ function getPluginManifest() {
   manifest.parent = { grouparoo: parentPkg.grouparoo, path: parentPath };
 
   // plugins
-  if (manifest.parent.grouparoo && manifest.parent.grouparoo.plugins) {
-    for (const i in manifest.parent.grouparoo.plugins) {
-      const pluginName = manifest.parent.grouparoo.plugins[i];
-
-      if (pluginName === "@grouparoo/core") continue;
-
-      let pluginPath = "";
-      try {
-        pluginPath = require.resolve(pluginName);
-      } catch {
-        pluginPath = path.join(parentPath, "node_modules", pluginName);
-        if (!fs.existsSync(pluginPath)) {
-          pluginPath = path.join(
-            grouparooMonorepoApp
-              ? path.join(
-                  __dirname,
-                  "..",
-                  "..",
-                  "..",
-                  "apps",
-                  grouparooMonorepoApp
-                )
-              : path.join(__dirname, "..", "..", "..", "..", "..", ".."),
-            "node_modules",
-            pluginName
-          );
-        }
-      }
+  let pluginNames = [...(manifest?.parent?.grouparoo?.plugins || [])];
+
+  const availableUiPlugins = [
+    "@grouparoo/ui-enterprise",
+    "@grouparoo/ui-community",
+    "@grouparoo/ui-config",
+  ];
+
+  const installedPackages = Object.keys(
+    parentPkg?.devDependencies || []
+  ).concat(Object.keys(parentPkg?.dependencies || []));
```

Comment:
Here's where I'm not sure you can be sure `devDependencies` are installed or not.  You might need to test if they exist with a `require.resolve() + fs` check

Reply:
Very good point. This was dumb because it should only be looking for the UI packages here. However, all this is doing is collecting them into an array. Lower in the file it is already checking to make sure they are actually installed, but I think could use a tighter check. I'll give this one another shot.